utils/docker.go

```diff
@@ -284,7 +288,19 @@ func RemoveNetwork(network types.NetworkResource) {
 	}
 }
 
-// GetPort will return the current port that PFE is running on (hardcoded to 9090 for now)
-func GetPort() string {
-	return "9090";
+// GetPFEPort will return the current port that PFE is running on (hardcoded to 9090 for now)
```

Comment:
`(hardcoded to 9090 for now)` doesnt need to be in this comment anymore

Reply:
Good spot! Fixed